UI/aoc-app/src/Components/CreateProfile.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+
+const CreateProfile = () => {
+    return (
+        <Form>
+            <FormGroup row>
+            <Label for="name" sm={2}>Complete Profile</Label>
+            </FormGroup>
+            <FormGroup row>
+                <Label for="name" sm={2}>Name</Label>
+                <Col sm={10}>
+                <Input type="text" name="name" id="name" placeholder="John Doe" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label for="email" sm={2}>Email</Label>
+                <Col sm={10}>
+                <Input type="email" name="email" id="email" placeholder="johndoe@gmail.com" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label for="phone" sm={2}>Phone Number</Label>
+                <Col sm={10}>
+                <Input type="tel" name="phone" id="phone" placeholder="415-879-7877" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label for="govId" sm={2}>Government Issued ID</Label>
+                <Col sm={10}>
+                <Input type="file" name="govId" id="govId" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label sm={2}>Reference 1:</Label>
+                <Label for="referenceName" sm={1}>Name</Label>
+                <Col sm={4}>
+                <Input type="text" name="referenceName" id="referenceName" />
+                </Col>
+                <Label for="referenceEmail" sm={1}>Email</Label>
+                <Col sm={4}>
+                <Input type="email" name="referenceEmail" id="referenceEmail" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label sm={2}>Reference 2:</Label>
+                <Label for="referenceName" sm={1}>Name</Label>
+                <Col sm={4}>
+                <Input type="text" name="referenceName" id="referenceName" />
+                </Col>
+                <Label for="referenceEmail" sm={1}>Email</Label>
+                <Col sm={4}>
+                <Input type="email" name="referenceEmail" id="referenceEmail" />
+                </Col>
+            </FormGroup>
+            <FormGroup row>
+                <Label for="tools" sm={2}>Tools/Supplies</Label>
+                <Col sm={10}>
+                <Input type="file" name="tools" id="tools" />
+                </Col>
+            </FormGroup>
+            <FormGroup check row>
+                <Col sm={{ size: 10, offset: 6 }}>
+                <Button>Submit</Button>
+                </Col>
+            </FormGroup>
+        </Form>
+    );
```

Comment:
This form overall looks good and aligns with the DB instantiated. Looking at this, made me think maybe we need a more general 'File Upload' section where multiple files can be uploaded and the cleaner has freedom over whether it's a resume, coverletter, whatever it may be. That way, we don't have to manually create a section for every type of file we anticipate and have an all-encompassing section. The DB is compatible with this type of design. But this is something to hash out further with our client, so for sprint 1 this looks good!

Reply:
Absolutely, it feels weird to ask for files with tools/supplies. This is why I originally questioned this in the very early requirements. I'm not a huge fan of it but I followed the mock-ups. We can very easily change this in sprint 2. Let's discuss in our next sprint planning.

Reply:
Good call out - this is definitely something we need to refine with Arelli Cleaning but I think this a great MVP. I will add this as an agenda item for our sprint planning meeting :)